examples/rhino-sdk/federated_join/Federated Join Tutorial.ipynb

```diff
@@ -0,0 +1,585 @@
+    "    join_field={\"data_column\": \"UID\", \"filter_column\": \"Age\", \"filter_value\": 35, \"filter_type\": \">\"},\n",
+    ")\n",
+    "\n",
+    "joined_results = session.project.joined_cohort_metric(\n",
+    "    filter_cohorts=[filter_cohort_at_lahey.uid],\n",
+    "    data_cohorts=[first_data_cohort_at_mgh.uid],\n",
+    "    configuration=configuration\n",
+    ")\n",
+    "\n",
+    "federated_result = joined_results.output['mean']\n",
+    "print(f\"Federeated Mean: {federated_result}\")\n",
+    "\n",
+    "a = filter_df[\"Age\"] > 35\n",
+    "b = first_data_df[\"Gender\"] == \"m\"\n",
+    "centralized_result = np.mean(first_data_df.SpO2[a&b])\n",
+    "print(f\"Centralized Mean: {centralized_result}\")\n",
+    "print(centralized_result == federated_result)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 99,
+   "id": "dd9dac9c",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Works with grouping. SpO2 values for both genders >35 years old\n",
+      "{'m': {'mean': 1.0224691358024691}, 'f': {'mean': 1.02375}}\n",
+      "True\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(\"Works with grouping. SpO2 values for both genders >35 years old\")\n",
+    "\n",
+    "configuration = Mean(\n",
+    "    variable=\"SpO2\",\n",
+    "    join_field={\"data_column\": \"UID\", \"filter_column\": \"Age\", \"filter_value\": 35, \"filter_type\": \">\"},\n",
+    "    group_by={\"groupings\": [\"Gender\"]},\n",
+    ")\n",
+    "\n",
+    "joined_results = session.project.joined_cohort_metric(\n",
+    "    filter_cohorts=[filter_cohort_at_lahey.uid],\n",
+    "    data_cohorts=[first_data_cohort_at_mgh.uid],\n",
+    "    configuration=configuration\n",
+    ")\n",
+    "\n",
+    "print(joined_results.output)\n",
+    "\n",
+    "c = first_data_df[\"Gender\"] == \"f\"\n",
+    "female_centralized_result = np.mean(first_data_df.SpO2[a&c])\n",
+    "print(female_centralized_result == joined_results.output['f']['mean'])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 100,
+   "id": "ede912ee",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Filtering on multiple data columns\n",
+      "{'mean': 1.0492307692307692}\n",
+      "1.0492307692307692\n",
+      "True\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(\"Filtering on multiple data columns\")\n",
+    "\n",
+    "configuration = Mean(\n",
+    "    variable=\"SpO2\",\n",
+    "    join_field=\"UID\",\n",
+    "    data_filters=[\n",
+    "        {\n",
+    "            \"filter_column\": \"Age\",\n",
+    "            \"filter_value\": 35,\n",
+    "            \"filter_type\": \">\",\n",
+    "            # For intersection mode, if unspecified apples to both cohorts. \n",
```

Comment:
I don't know what this means; could you elaborate more? If what is unspecified? Maybe provide a dueling example if you think it's an important enough call out